src/main/java/com/zaxxer/hikari/pool/ProxyConnection.java

```diff
@@ -455,11 +455,14 @@ private static Connection getClosedConnection()
             public Object invoke(Object proxy, Method method, Object[] args) throws Throwable
             {
                final String methodName = method.getName();
-               if ("abort".equals(methodName)) { 
+               if ("isValid".equals(methodName)) {
+                  return Boolean.FALSE;
+               }
+               else if ("close".equals(methodName)) { 
```

Comment:
not 'literally' consecutive calls like close after close.
people use 'api' in ways author not intended for :)

It may happen... for some reason... unwrapped connection be held somewhere in code then later some time close would be called on it.